### PR TITLE
create issue templates

### PR DESCRIPTION
**Pull request by horrible-little-slime**

### CODE REVIEW

**Comment by horrible-little-slime** [comment]
oh we should add a little checkbox that says "I have read the garbo bug reporting procedure" that points to some other markdown file that contains a better version of the pin, and make users check that box before submitting an issue.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve buggy behavior
+title: "[BUG]"
+labels: ""
+assignees: ""
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is. Be sure that the behavior you're describing _is_ a bug--garbo may do things you don't understand, but not understanding something isn't a reason to report it. You should also take that advice into your life outside KoL; the post-9/11 security state has taught is to be wary of our friends and neighbors in a way that is ultimately detrimental to society.
```
Comment (burningbman) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: Do we want snark in a template?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Session Log**
+Attach a full KoLMafia session log from the day you encountered the bug. Include details about when in the log the bug occurred to help developers rapidly find the error. Copy and paste any associated red text printed to the gCLI during this error.
+
+**KoLMafia Version**
+Ensure that KoLMafia and garbo are both up to date before reporting a bug. If you have installed mafia in a weird or non-standard way, let us know any details about that here. But also, if you've installed mafia in a weird or non-standard way, I'm telling you right now that that's your bug.
```
Comment (burningbman) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: Move this to the very top in a "Before Reporting" section?

File: README.md [hunk 3]
```diff
@@ -138,7 +138,7 @@ Garbo can run for a specified number of turns, or run until you have a specified
 
 Running Garbo `help` will print out simple help documentation and not run any turns.
 
-If you have issues with this script, please post about them in the #garbage-collector channel within the [Ascension Speed Society Discord](https://discord.gg/tbUCRT5), and someone will eventually (at some point) endeavor to solve them. Someday. Maybe.
+If you have issues with this script, please make a github issue!
```
Comment (burningbman) [inline README.md:141]: Link to the issues page?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (horrible-little-slime) [reply .github/ISSUE_TEMPLATE/bug_report.md:13]: This isn't intended as snark, but it probably doesn't belong nevertheless.

**Pull request merged**